### PR TITLE
[Commands] Cleanup #ban, #ipban, #flag, #kick, #setlsinfo, and #setpass Commands.

### PR DESCRIPTION
**Pull request by Kinglykrab**
- Cleanup messages and logic.
- Add ServerFlagUpdate_Struct for flag updates.
- Cleanup quest::whisper to use Chat::EchoChat1 instead of 315.

### CODE REVIEW

File: common/database.cpp [hunk 1]
```diff
@@ -92,112 +92,123 @@ Database::~Database()
...
 	return false;
 }
 
-bool Database::AddBannedIP(char* bannedIP, const char* notes) {
-	std::string query = StringFormat("INSERT into banned_ips SET ip_address='%s', notes='%s'", bannedIP, notes);
+bool Database::AddBannedIP(std::string banned_ip, std::string notes) {
+	auto query = fmt::format(
+		"INSERT into banned_ips SET ip_address = '{}', notes = '{}'",
+		banned_ip,
+		notes
```
Comment (Akkadius) [inline common/database.cpp:150]: Should probably escape these

File: zone/gm_commands/setlsinfo.cpp [hunk 2]
```diff
@@ -5,20 +5,19 @@ extern WorldServer worldserver;
...
-		worldserver.SendPacket(pack);
-		c->Message(Chat::White, "Login Server update packet sent.");
+	int arguments = sep->argnum;
+	if (arguments < 2) {
+		c->Message(Chat::White, "Format: #setlsinfo [Email] [Password]");
+		return;
 	}
+
+	auto pack = new ServerPacket(ServerOP_LSAccountUpdate, sizeof(ServerLSAccountUpdate_Struct));
+	ServerLSAccountUpdate_Struct *s = (ServerLSAccountUpdate_Struct *) pack->pBuffer;
```
Comment (Akkadius) [inline zone/gm_commands/setlsinfo.cpp:15]: Can auto

File: zone/gm_commands/setlsinfo.cpp [hunk 3]
```diff
@@ -5,20 +5,19 @@ extern WorldServer worldserver;
...
-		ServerLSAccountUpdate_Struct *s   = (ServerLSAccountUpdate_Struct *) pack->pBuffer;
-		s->useraccountid = c->LSAccountID();
-		strn0cpy(s->useraccount, c->AccountName(), 30);
-		strn0cpy(s->user_email, sep->arg[1], 100);
-		strn0cpy(s->userpassword, sep->arg[2], 50);
-		worldserver.SendPacket(pack);
-		c->Message(Chat::White, "Login Server update packet sent.");
+	int arguments = sep->argnum;
+	if (arguments < 2) {
+		c->Message(Chat::White, "Format: #setlsinfo [Email] [Password]");
```
Comment (Akkadius) [inline zone/gm_commands/setlsinfo.cpp:10]: If you want to be consistent with other commands `Usage:` vs `Format:`

Comment (Kinglykrab) [reply zone/gm_commands/setlsinfo.cpp:10]: 👍

**Review by Akkadius — APPROVED** [review]
Assuming thoroughly tested, lgtm

**Comment by Kinglykrab** [comment]
> Assuming thoroughly tested, lgtm

Yeah, some commands were added later than others, as they fit the scope of the pull request.

**Pull request merged**